bin/src/types.js

```diff
@@ -9,6 +9,11 @@ const Uuid = t.refinement(t.String, isUUID, 'UUID');
 
 const Test = t.struct({
   title: t.String,
+  trid: t.maybe(t.String),
```

Comment:
Add comments to specify Airware specific things. Start with begin and end

// start aw
...
// end aw